### ISSUE TITLE
Can we add new games?

### ISSUE THREAD

**Issue by dfsdfsdfs** [floatinghotpot/casino-server#14]
Can we add our games to this platform in an easy way?

**Comment by nitinmukesh** [comment]
Yes it is possible if the game is multi-room, multi/single player.
In casino.conf.js you can add new game type and define the path of JS file with game logic
